consensus/quorum/one-node-one-vote.go

```diff
@@ -43,6 +63,13 @@ func (v *uniformVoteWeight) IsRewardThresholdAchieved() bool {
 	return v.SignersCount(Commit) >= (v.ParticipantsCount() * 9 / 10)
 }
 
+// ComputeTotalPowerByMask computes the total power indicated by bitmap mask
+func (v *uniformVoteWeight) computeTotalPowerByMask(mask *bls_cosi.Mask) *numeric.Dec {
```

Comment:
Use int here. Don't use Dec for uniform vote

Reply:
removed this function